packages/core/src/components/section/section.tsx

```diff
@@ -158,38 +167,37 @@ export const Section: React.FC<SectionProps> = React.forwardRef((props, ref) =>
             })}
             elevation={elevation}
             ref={ref}
+            role="region"
```

Comment:
I appreciate the attempt to provide better markup out-of-the-box, but after looking at the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role) for this aria role, I don't think it's appropriate to add it by default to _every_ Section component. This component is too generic for that, its purpose is largely for styling rather than functionality. So we should leave it up to users to specify `role="region"` via `{...htmlProps}` if they want.

```suggestion
```

Reply:
done